com.unity.render-pipelines.universal/Editor/ShaderGraph/Templates/ShaderPass.template

```diff
@@ -114,7 +118,12 @@ $splice(GraphPixel)
 
 // --------------------------------------------------
 // Build Graph Inputs
-
+#ifdef HAVE_VFX_MODIFICATION
+#define VFX_SRP_ATTRIBUTES Attributes
```

Comment:
So nice to see this abstracted between pipelines!

Reply:
It was quite pleasing to end with a simple abstraction like this.
There is only one difference which remains: the `fragInputs` design is specific to HDRP and there is some spefic code in `VFXCodeGenerator.cs` to handle this indirection.